specification/communication/data-plane/Chat/preview/2022-11-15-preview8/communicationserviceschat.json

```diff
@@ -0,0 +1,1977 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Azure Communication Chat Service",
+    "description": "Azure Communication Chat Service",
+    "version": "2022-11-15-preview8"
```

Comment:
The version should be a date that is close to when you're actually going to release (so 2023) and we're not doing the number suffix after preview anymore